### ISSUE TITLE
Szószedet - Szótár készítése

### ISSUE THREAD

**Issue by ghost** [arphox/tudasbazis#21]
Érdemes lenne összeszedni a gyakran használt idegen/szakmai szavakat, hogy mi mit jelen, hogy egy helyen legyen minden ha valaki keres :)

Például:
- Backend
- Frontend
- Fullstack
- Interpreter
- Compiler 
éstatöbbi :)

Szerintetek?

**Comment by arphox** [comment]
Jó ötlet, de én csak úgy látom értelmét, ha mindent a lehető legrövidebben magyarázunk meg. Meg akkor legyen mondjuk ABC-sorrendben. Többiek?

**Comment by ghost** [comment]
Igen, ilyen nagyon maximum pár soros, pár mondatos dolgokra gondoltam. Azért nem kell 1 mondatba sűríteni, de jogos, hogy ne hosszan taglalva mély részletekig leírva legyen :)
